notebooks/Tutorial3_Embedded_processors.ipynb

```diff
@@ -221,11 +221,11 @@
     "\n",
     "To reduce the runtime caused by the imputation, it is possible to use an alternative imputation such as:\n",
     "- `ImputeMethod.fast_knn`, an appromixation of a knn imputer\n",
-    "- `ImputeMethod.mean` that imputes using the mean of each variable (or mode if non-numeric)\n",
-    "- `ImputeMethod.mode` that imputes using the mode of each variable\n",
-    "- `ImputeMethod.median` that imputes using the median of each variable\n",
+    "- `ImputeMethod.mean` which imputes using the mean of each variable (or mode if non-numeric)\n",
+    "- `ImputeMethod.mode` which imputes using the mode of each variable\n",
+    "- `ImputeMethod.median` which imputes using the median of each variable\n",
```

Comment:
I believe they should be `that`
-> from google: "Remember that “which” is always preceded by a comma, while “that” is never preceded by a comma"